src/components/timeline/TimelineMarkerCanvas.js

```diff
@@ -259,7 +258,7 @@ class TimelineMarkerCanvas extends PureComponent {
     const minDuration =
       rangeLength *
       viewportLength *
-      (rowHeight * 2 * MARKER_DOT_RADIUS / containerWidth);
+      (rowHeight * 2 * MARKER_MIN_DOT_RADIUS / containerWidth);
```

Comment:
I tried with this value and it looks OK to me; with MARKER_MAX_DOT_RADIUS the behavior is worse because it starts to hit too far from the marker.